imagevector/containers.yaml

```
@@ -843,26 +843,28 @@ images:
             teamname: 'gardener/gardener-core-networking-maintainers'
       - name: 'cloud.gardener.cnudie/dso/scanning-hints/package-versions'
         value:
-          # https://www.envoyproxy.io/docs/envoy/v1.29.2/intro/arch_overview/security/external_deps
+          # https://www.envoyproxy.io/docs/envoy/v1.35.0/intro/arch_overview/security/external_deps
           - name: 'v8'
-            version: '10.7.193.13'
+            version: '13.8.258.26'
           - name: 'grpc'
-            version: '1.59.4'
+            version: '1.72.0'
           - name: 'luajit'
-            version: '2.1.0.beta3'
+            version: '19878ec'
           - name: 'nghttp2'
-            version: '1.58.0'
+            version: '1.72.0'
```

`grpc` has been updated to `1.72.0`, but for `nghttp2` the [page](https://www.envoyproxy.io/docs/envoy/v1.35.0/intro/arch_overview/security/external_deps) shows:
```suggestion
            version: '1.66.0'
```